mapreduce/invertedindex.go

```diff
@@ -25,62 +25,68 @@ func Map(input MRInput, resultChl chan MRInput, doneChl chan bool) {
 	for _, word := range words {
 		resultChl <- MRInput{Key: word, Values: []string{fileName}}
 	}
-	doneChl <- true
+	doneChl <- struct{}{}
 }
 
 // RemoveDups removes any duplicates from MRInput.Values. Specifically, if a given word appears more than once in a
 // file, then that file's name will appear multiple times in MRInput.Values after processing by the Map() function.
-func RemoveDups(input MRInput, resultChl chan MRInput, doneChl chan bool) {
-	set := make(map[string]bool)
+func RemoveDups(input MRInput, resultChl chan MRInput, doneChl chan struct{}) {
+	set := make(map[string]struct{})
```

Comment:
Can you change `map[string]struct{}` back to `map[string]bool`. I believe the latter is more idiomatic.